script/entrypoint.sh

```diff
@@ -1,20 +1,42 @@
 #!/usr/bin/env bash
 
 CMD="airflow"
-TRY_LOOP="10"
-POSTGRES_HOST="postgres"
-POSTGRES_PORT="5432"
-RABBITMQ_HOST="rabbitmq"
-RABBITMQ_CREDS="airflow:airflow"
-FERNET_KEY=$(python -c "from cryptography.fernet import Fernet; FERNET_KEY = Fernet.generate_key().decode(); print FERNET_KEY")
+TRY_LOOP="${TRY_LOOP:-10}"
+POSTGRES_HOST="${POSTGRES_HOST:-postgres}"
+POSTGRES_PORT=5432
+POSTGRES_CREDS="${POSTGRES_CREDS:-airflow:airflow}"
+RABBITMQ_HOST="${RABBITMQ_HOST:-rabbitmq}"
+RABBITMQ_CREDS="${RABBITMQ_CREDS:-airflow:airflow}"
+RABBITMQ_MANAGEMENT_PORT=15672
+FLOWER_URL_PREFIX="${FLOWER_URL_PREFIX:-}"
+AIRFLOW_URL_PREFIX="${AIRFLOW_URL_PREFIX:-}"
+LOAD_DAGS_EXAMPLES="${LOAD_DAGS_EXAMPLES:-true}"
+GIT_SYNC_REPO="${GIT_SYNC_REPO:-}"
+
+if [ -z $FERNET_KEY ]; then
+    FERNET_KEY=$(python3 -c "from cryptography.fernet import Fernet; FERNET_KEY = Fernet.generate_key().decode(); print(FERNET_KEY)")
+fi
+
+echo "Postgres host: $POSTGRES_HOST"
+echo "RabbitMQ host: $RABBITMQ_HOST"
+echo "Load DAG examples: $LOAD_DAGS_EXAMPLES"
+echo "Git sync repository: $GIT_SYNC_REPO"
+echo
 
 # Generate Fernet key
-sed -i "s/{FERNET_KEY}/${FERNET_KEY}/" $AIRFLOW_HOME/airflow.cfg
+sed -i "s/{{ FERNET_KEY }}/${FERNET_KEY}/" $AIRFLOW_HOME/airflow.cfg
+sed -i "s/{{ POSTGRES_HOST }}/${POSTGRES_HOST}/" $AIRFLOW_HOME/airflow.cfg
+sed -i "s/{{ POSTGRES_CREDS }}/${RABBITMQ_CREDS}/" $AIRFLOW_HOME/airflow.cfg
```

Comment:
This looks like a typo.  It breaks when `POSTGRES_CREDS` != `RABBITMQ_CREDS`.

Reply:
ho yes

Reply:
should be fixed now